### ISSUE TITLE
Keeping FanControl Updated & Minimised

### ISSUE THREAD

**Issue by leeramsay** [Rem0o/FanControl.Releases#1781]
I use my PC as a Gaming Console, it boots into Steam Big Picture mode, and I primarily control everything using a controller. I want FanControl to start minimised and do it's thing.

When a new update is released, FanControl notifies me, and I need a keyboard and mouse to action/return to what I was doing.

I tried to adjust to launch Updater.exe, which launches FanControl after it's done, but this behaviour seemingly ignores the 'start minimised' option I have in my configuration, which thus opens FanControl and requires a keyboard and mouse to minimise.

My ideal scenario would be to check for an update each launch, update if necessary, and then launch minimised, with no user input.

I tried to see if there's any command line arguments I could leverage to achieve my desired behaviour, but I couldn't find any documented switches for Updater.exe, and the [FanControl command line options](https://getfancontrol.com/docs/) doesn't currently include anything that would help.

So in summary, it seems like my options at this point in time are:
1. Launch FanControl, as opposed to updater, and get prompted for updates occasionally
2. Use Updater.exe, and have to minimise each time
3. Write my own code to Launch Updater.exe, wait until a fan control process is started, and then kill this process and relaunch FanControl (which does honour the start minimised configuration)

Is there an existing solution that I've missed? Or could Updater.exe be updated so it honours the minimise flag? Or atleast expose a switch to allow me to control the behaviour?

Thank you in advance :)

**Comment by Rem0o** [comment]
I guess the updater could receive the -m flag and forward it to FanControl. However I wouldn't recommend using the updater to launch FanControl each time.

What I would do if I were you since it's a gaming full picture box, is once you are satisfied with FanControl:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/11cecb5f-15b2-432f-b47d-ebbfa458ec63)

And live on that version until you remember FanControl is even running on that computer. No more update popups, and a stable FanControl running in the background.

**Comment by FireStryker** [comment]
I read that tick box as "don't bug me again for this version" wouldn't that pop up again when the next version is pushed?

**Comment by Rem0o** [comment]
Once you update, the checkbox resets. 

So if you want to stay on a specific version forever, tick it, that's what it is meant for. See it as a "lock" for the installed version.

**Comment by leeramsay** [comment]
I will put everything back how it was and test @Rem0o thanks, but I too interpreted it as (just for example, let’s say I’m on v1, and I have an update waiting for v2): “tick this box to disable updates until v3”.

I’m sure I’ve checked it and then later seen another pop up with no further changes on my side to my FanControl config, but I’ll double check.
